application/models/thread.go

```diff
@@ -116,6 +119,16 @@ func (t *Thread) LoadParticipants(tx *pop.Connection) error {
 	if err := tx.Load(t, "Participants"); err != nil {
 		return fmt.Errorf("error loading threads participants %v ... %v", t.ID, err)
 	}
+
+	return nil
+}
+
+func (t *Thread) LoadThreadParticipants(tx *pop.Connection) error {
+
+	if err := tx.Load(t, "ThreadParticipants"); err != nil {
```

Comment:
Does this work? There isn't a `ThreadParticipants` field on `Thread`.

Reply:
I think I did it this way because of your previous comment about making use of the many-to-many relationship.

It doesn't look like it's being called now.  I'll check if we need it.